SCClassLibrary/Common/Core/Object.sc

```diff
@@ -137,6 +137,16 @@ Object  {
 		^this.performWithEnvir(selector, ().putPairs(pairs))
 	}
 
+	performWith {|selector, argumentsArray=([]), keywordArgumentEnvir=(())|
```

Comment:
default arguments are given in `makePerformableArray` so `argumentsArray=([])` can simply be `argumentsArray`. This will be more efficient. Same for `keywordArgumentEnvir`.